prow/cmd/plank/main.go

```diff
@@ -39,12 +39,14 @@ var (
 	totURL = flag.String("tot-url", "", "Tot URL")
 
 	configPath   = flag.String("config-path", "/etc/config/config", "Path to config.yaml.")
+	cluster      = flag.String("cluster", "", "Path to kube.Cluster YAML file. If empty, uses the local cluster.")
```

Comment:
We may want to sync [`--build-cluster`](https://github.com/kubernetes/test-infra/blob/1e5e44f72851b6a3e5d9db1dcb1cd76c40a19f8a/prow/cmd/deck/main.go#L45) with this in a follow-up. (or the other way around?)

Reply:
Yes.